Makefile

```diff
@@ -16,6 +16,18 @@ render:
 	./scripts/render-docs-for-components.sh
 	./scripts/render-docs-for-modules.sh
 	./scripts/render-docs-for-github-actions.sh
+	
+containers:
+	docker build . -t cloudposse/docs
+	docker build scripts -t cloudposse/docs:scripts
+
+docker/render: containers
+	docker run --rm -v -it $(PWD):/app -w /app cloudposse/docs:scripts scripts/render-docs-for-components.sh
+	docker run --rm -v -it $(PWD):/app -w /app -e PUBLIC_REPO_ACCESS_TOKEN="$$(gh auth token)" cloudposse/docs:scripts scripts/render-docs-for-modules.sh
+	docker run --rm -v -it $(PWD):/app -w /app -e PUBLIC_REPO_ACCESS_TOKEN="$$(gh auth token)" cloudposse/docs:scripts scripts/render-docs-for-github-actions.sh
+
+docker/build: containers
+	docker run --rm -v $(PWD):/app -w /app cloudposse/docs npm run build
```

Comment:
Does this work now? That would be awesome

Reply:
@Benbentwo tested it, I believe. I'm not sure if Dan did. @milldr were you able to try out the container version?

Reply:
I haven't, but if it works for you both I'll take your word for it